portfolio/src/main/webapp/backpacking.html

```diff
@@ -6,19 +6,139 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <div id="content">
-      <h1>Jimmy's Backpacking Adventures</h1>
-      <div class="image-thumbnails">
-        <a href="images/backpacking/bp-1.JPG"><img src="images/backpacking/bp-1.JPG" width="504" height="378"/></a>
-        <a href="images/backpacking/bp-2.JPG"><img src="images/backpacking/bp-2.JPG" width="504" height="378"/></a>
-        <a href="images/backpacking/bp-4.JPG"><img src="images/backpacking/bp-4.JPG" width="504" height="378"/></a>
-        <a href="images/backpacking/bp-5.JPG"><img src="images/backpacking/bp-5.JPG" width="504" height="378"/></a>
-        <a href="images/backpacking/bp-9.jpg"><img src="images/backpacking/bp-9.jpg" width="504" height="378"/></a>
-        <a href="images/backpacking/bp-7.jpg"><img src="images/backpacking/bp-7.jpg" width="504" height="378"/></a>
-        <a href="images/backpacking/bp-8.jpg"><img src="images/backpacking/bp-8.jpg" width="504" height="378"/></a>
-        <a href="images/backpacking/bp-6.JPG"><img src="images/backpacking/bp-6.JPG" width="504" height="378"/></a>
-        <a href="images/backpacking/bp-10.jpg"><img src="images/backpacking/bp-10.jpg" width="504" height="378"/></a>
+
+    <div id="navbar">
+      <a href="index.html">Home</a>
+      <a href="backpacking.html">Backpacking</a>
+      <a href="random.html">Humans I Love</a>
+    </div>
+
+    <div class="slideshow-container">
+      <div class="mySlides">
+        <div class="numbertext">1 / 10</div>
+          <img src="images/backpacking/bp-1.JPG" style="width:100%">
+      </div>
+      <div class="mySlides">
+        <div class="numbertext">2 / 10</div>
+          <img src="images/backpacking/bp-2.JPG" style="width:100%">
+      </div>
+      <div class="mySlides">
```

Comment:
Did we intentionally skip 3?

Reply:
Yes, image 3 had different dimensions than the others and messed up the neatness of the presentation.